contracts/protocol/PositionManager.sol

```diff
@@ -675,7 +675,6 @@ contract PositionManager is
                         state.remainingSize = 0;
                         state.pip = step.pipNext;
                         isFullBuy = 0;
-                        passedPipCount++;
```

Comment:
when order fully filled liquidity in a pip but still stay in the same pip, passed pip count should not be increased cause it didn't pass any pip